config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
   root 'locations#index'
   devise_for :users
-  resources :locations, only: [:index, :new, :show]
+  resources :locations, only: [:index, :new, :show] do
+    resources :reviews, only: []
```

Comment:
I'm not sure what this is doing for you? Please delete it, unless I'm just being dense 😁